contracts/lib/Approvable.sol

```diff
@@ -3,18 +3,19 @@ pragma solidity ^0.4.11;
 import './Ownable.sol';
 
 contract Approvable is Ownable {
-  mapping(address => bool) approved;
+  mapping(address => bool) public approved;
 
   function Approvable() {
-    approveAddress(msg.sender);
+    updateAddressState(msg.sender, true);
```

Comment:
Yes, yes it will.  Hmm, let me think about possible solutions to this.  One idea is to allow adding an address if the `approved` mapping was empty, but then later if we had removed all addresses anyone could add one.  Will think more about this and come back to it.